### ISSUE TITLE
wsgidav and CherryPy installed from PyPI: ImportError: cannot import name wsgiserver

### ISSUE THREAD

**Issue by ackalker** [mar10/wsgidav#65]
Using Python 2.7.13, Void Linux x86_64. Installed all of `wsgidav`, `lxml` and `cherrypy` using:
```sh
$ pip install --user wsgidav
$ pip install --user lxml
$ pip install --user cherrypy
```
A snippet from the output of `pip freeze`:
```
CherryPy==10.1.0
WsgiDAV==2.1.0
lxml==3.7.2
```
When I run `wsgidav`, I get the following error:
```
$ wsgidav --host=0.0.0.0 --port=80 --root=/tmp
WARNING: share '/' will allow anonymous access.
WARNING: Could not import lxml: using xml instead (slower).
         Consider installing lxml https://pypi.python.org/pypi/lxml.
******************************************************************************
ERROR: Could not import CherryPy.
Try `pip install cherrypy` or specify another server using the --server option.
******************************************************************************
Traceback (most recent call last):
  File "/home/miki/.local/bin/wsgidav", line 11, in <module>
    load_entry_point('WsgiDAV==2.1.0', 'console_scripts', 'wsgidav')()
  File "/home/miki/.local/lib/python2.7/site-packages/wsgidav/server/run_server.py", line 499, in run
    handler(app, config, server)
  File "/home/miki/.local/lib/python2.7/site-packages/wsgidav/server/run_server.py", line 352, in _runCherryPy
    from cherrypy import wsgiserver
ImportError: cannot import name wsgiserver
```
(Note the warning about `lxml`, which *is* installed locally, as are its system library dependencies. I can import it manually without problems.)

There is no `cherrypy.wsgiserver` package in my installation of `cherrypy`. From CherryPy's [`CHANGES.rst`](https://github.com/cherrypy/cherrypy/blob/master/CHANGES.rst), I gather:
> v9.0.0
> 
> *  #1481: Move functionality from cherrypy.wsgiserver to the cheroot 5.0 project.

Could you please support this new [`cheroot`](https://pypi.org/project/Cheroot/5.0.1/) project, or provide an upper bound for the version of `cherrypy` still supported by `wsgidav`? Thanks in advance.

**Comment by mar10** [comment]
- lxml is not recognized due to a regression (introduction of defusedxml)
- we could add a new option `--server=cheroot` (default) and rename `--server=cherrypy` to 'cherrypy-wsgiserver'
- Add support for OpenSSL (New option `--ssl-adapter`), currently blocked on cherrypy/cheroot#6

**Comment by mar10** [comment]
Fixed with v2.2.1
